AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -124,35 +125,46 @@ def AtomEditorComponents_Light_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, light_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # 5. Cycle through all light types.
+        for ltype, value in LIGHT_TYPES.items():
```

Comment:
you can probably use .keys() since you aren't using value so:
for light_type in LIGHT_TYPES.keys()
also ltype should probably be light_type as a variable name

Reply:
Done, yeah went with ltype because "light type" is used in other areas of the script, but no prob